internals/secrethub/tree.go

```diff
@@ -87,23 +90,29 @@ func (cmd *TreeCommand) printTree(t *api.Tree, w io.Writer) {
 
 // printDirContentsRecursively is a recursive function that prints the directory's contents
 // in a tree-like structure, subdirs first followed by secrets.
-func printDirContentsRecursively(dir *api.Dir, prefix string, w io.Writer, format [4]string) {
+func (cmd *TreeCommand) printDirContentsRecursively(dir *api.Dir, prefix string, w io.Writer, format [4]string, prevPath string) {
 
 	sort.Sort(api.SortDirByName(dir.SubDirs))
 	sort.Sort(api.SortSecretByName(dir.Secrets))
 
 	total := len(dir.SubDirs) + len(dir.Secrets)
 
+	if !cmd.fullPaths {
+		prevPath = ""
+	} else {
+		prevPath += "/"
+	}
```

Comment:
Nitpicky level high:

In case where we're dealing with both the truethy and the falsy value of a variable (if/else), I usually find it more readable not the use negation, to prevent the else case from being a negation of a negation (not not full paths).

How would you feel about changing to the following?

```suggestion
	if cmd.fullPaths {
		prevPath += "/"
	} else {
		prevPath = ""
	}
```

Reply:
Good point! Onto it.

Reply:
This should now be fixed.